Msg/Msg.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity >= 0.6.0;
+
+struct SendStatus {
+	bool succeeded; 
+	int32 code;
+}
+
+interface IMsg {
+
+    function send(uint32 answerId, TvmCell message, bool defSigner) external returns (SendStatus status);
```

Comment:
I think we should add "string msg2user" field. It is some message to user. This field describes what the user wants to sign and which key pair should be used.
function send(uint32 answerId, TvmCell msg2bchn, string msg2user, bool defSigner) external returns (SendStatus status);